hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/MiniDFSCluster.java

```diff
@@ -928,6 +928,11 @@ private void initMiniDFSCluster(
           dnStartOpt != null ? dnStartOpt : startOpt,
           racks, hosts, storageCapacities, simulatedCapacities, setupHostsFile,
           checkDataNodeAddrConfig, checkDataNodeHostConfig, dnConfOverlays);
+      try{
+        Thread.currentThread().sleep(1000);
+      } catch (Exception e) {
+        LOG.error("sleep error");
+      }
```

Comment:
Is this modification avoidable? It is better to keep this class unchanged. This modification has an influence on many test classes.